src/python/pants/backend/jvm/subsystems/scala_platform.py

```diff
@@ -152,22 +151,14 @@ def _memoized_scalac_classpath(self, scala_path, scheduler):
     if scala_path:
```

Comment:
This will now return `None` if there is no `scala_path`... intentional? Would be good to be explicit probably.